src/test/java/uk/gov/hmcts/cmc/claimstore/constraints/DateNotInThePastValidatorTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.cmc.claimstore.constraints;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.time.LocalDate;
+import javax.validation.ConstraintValidatorContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DateNotInThePastValidatorTest {
+
+    @Mock
+    private ConstraintValidatorContext context;
+
+    private DateNotInThePastConstraintValidator validator = new DateNotInThePastConstraintValidator();
+
+    @Test
+    public void shouldReturnTrueForValidDateInTheFuture() {
```

Comment:
@pragnesh-patel can you check you're happy with the cases covered here

Reply:
The same cases are written for DateNotInTheFuture. If this is not enough - both should be updated